### PR TITLE
opaque Program, Cmd.Of*

### PR DESCRIPTION
**Pull request by et1975**
These are the ideas so far, I'll try applying some of this in Elmish.Browser and see if/how it feels.

### CODE REVIEW

File: src/cmd.fs [hunk 1]
```diff
@@ -41,76 +37,203 @@ module Cmd =
...
-                task arg
-            with x ->
-                x |> (ofError >> dispatch)
-        [bind]
-
     /// Command to call the subscriber
     let ofSub (sub: Sub<'msg>) : Cmd<'msg> =
         [sub]
 
+    module OfFunc =
```
Comment (MangelMaxime) [inline src/cmd.fs]: Is `Of` adding value compare to naming the module `Func`?

Comment (et1975) [reply src/cmd.fs]: I think so - it is more discoverable by people who just start typing the same thing they are used to, and I'd want it in the name - function or module, the module seems preferable.

Comment (MangelMaxime) [reply src/cmd.fs]: Ok, thanks for the explanations.

**Comment by alfonsogarciacaro** [comment]
Is there any timeframe for releasing this? I'm planning to write a blog post announcing the changes in Fable.Core, Fable.Browser packages, etc. If Elmish API is going to change, it's probably a good idea to add information about that too, as it will likely affect most of Fable projects.

**Comment by et1975** [comment]
I should be able to wrap it up in the next couple of days. Debugger turns out to be quite an outlier in the conversion.

**Comment by et1975** [comment]
Released the changes along with Browser, React and Debugger. @MangelMaxime will you update the HMR?

**Comment by MangelMaxime** [comment]
@et1975 Yes, I will do it this weekend.

**Comment by alfonsogarciacaro** [comment]
No, you won't ;) https://github.com/elmish/hmr/pull/21

**Pull request merged**